src/hotspot/share/gc/serial/tenuredGeneration.hpp

```diff
@@ -92,8 +92,8 @@ class TenuredGeneration: public Generation {
   size_t capacity() const;
   size_t used() const;
   size_t free() const;
-  MemRegion used_region() const;
 
+  MemRegion used_region() const { return space()->used_region(); }
```

Comment:
This move requires inclusion of `space.hpp`.

Maybe it is better to move `save_used_region()` into the `.inline.hpp` file as well and include the `.inline.hpp` file where needed.